src/wallet/wallet.cpp

```diff
@@ -3083,6 +3083,7 @@ bool CWallet::GetNewDestination(const OutputType type, const std::string label,
     bool result = false;
     auto spk_man = m_spk_man.get();
     if (spk_man) {
+        spk_man->TopUp();
```

Comment:
In commit "keypool: Move opportunistic TopUps from LegacyScriptPubKeyMan to CWallet and ReserveDestination" (5921040b61bb5bdceddfbb94e58d51852f403232)

I think you forgot to remove corresponding topup from LegacyScriptPubKeyMan::GetNewDestination

Reply:
Oops. Fixed.